driver-opentelemetry/src/main/java/com/datastax/driver/opentelemetry/OpenTelemetryTracingInfoFactory.java

```diff
@@ -7,31 +7,41 @@
 import io.opentelemetry.api.trace.Tracer;
 import io.opentelemetry.context.Context;
 
+import static com.datastax.driver.opentelemetry.PrecisionLevel.NORMAL;
+
 public class OpenTelemetryTracingInfoFactory implements TracingInfoFactory {
   private final Tracer tracer; // OpenTelemetry Tracer object
+  private final PrecisionLevel precision;
 
   public OpenTelemetryTracingInfoFactory(final Tracer tracer) {
     this.tracer = tracer;
+    this.precision = NORMAL;
```

Comment:
You can use the other constructor instead of creating two different. I mean something like this:
```
public OpenTelemetryTracingInfoFactory(final Tracer tracer) {
    this(tracer, NORMAL);
}
```